bugshaker/src/main/java/com/github/stkent/bugshaker/ScreenshotUtil.java

```diff
@@ -0,0 +1,49 @@
+package com.github.stkent.bugshaker;
+
+import java.io.File;
+
+import android.app.Application;
+import android.content.Context;
+import android.support.annotation.NonNull;
+
+import com.github.stkent.bugshaker.flow.email.screenshot.BasicScreenShotProvider;
+import com.github.stkent.bugshaker.flow.email.screenshot.ScreenshotProvider;
+import com.github.stkent.bugshaker.flow.email.screenshot.maps.MapScreenshotProvider;
+import com.github.stkent.bugshaker.utilities.Logger;
+
+
+public class ScreenshotUtil {
+
+	private static final String SCREENSHOTS_DIRECTORY_NAME = "bug-reports";
+	private static final String SCREENSHOT_FILE_NAME = "latest-screenshot.jpg";
+	public static final String IMAGE_FILE_NAME = "Screenshot.png";
```

Comment:
screenshot_directory_name and screenshot_file_name refer to the screenshot that is being taken of the first activity, while image_file_name and image_description refer to the annotated image in the second activity